Anguandia/quickcredit
# Add delete user feature

#### What does this pull request do?

Add the delete user feature

#### Description of Task to be completed?

- Add functionality to the del function placeholder in the user controller
- Activate tests for the delete user route

#### How should this be manually tested?

- Signup on postman with a POST/api/v1/auth/signup with your names, email and isAdmin=true
- Copy the token in the response and update the postman client auth token
- Create another user as before
- Delete this new user with a delete/api/v1/users/<:userId> request
- The response should notify you of success in the deletion

#### What are the relevant pivotal tracker stories?

[finished #166083148]
